configs/projects/agent-runtime-5.5.x.rb

```diff
@@ -8,4 +8,7 @@
   # Dependencies specific to the 5.5.x branch
   proj.component 'rubygem-gettext-setup'
   proj.component 'rubygem-multi_json'
+  proj.component 'rubygem-highline'
+  proj.component 'rubygem-trollop'
+  proj.component 'rubygem-hiera-eyaml'
```

Comment:
Should this go in the master runtime too?

Reply:
Yeah I wasn't sure about that, asked about it just now.

Reply:
Typically we've been just applying component changes like this to all the branches they're supposed to be in at once -- but on the other hand, until this point the work we've been doing in this repo has been focused on migrating things that used to be in puppet-agent to this repo. That said, if everything builds and you can get some agent acceptance tests to pass, I wouldn't feel bad about adding these gems to both 5.5.x and master if no one else has a strong opinion.